src/Storage/Program.cs

```diff
@@ -242,11 +243,13 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     services.AddSingleton<ISasTokenProvider, SasTokenProvider>();
     services.AddSingleton<IKeyVaultClientWrapper, KeyVaultClientWrapper>();
     services.AddSingleton<IPDP, PDPAppSI>();
+    services.AddSingleton<IFileScanQueueClient, FileScanQueueClient>();
```

Comment:
We might be missing the line to adding the FileScanQueueSettings to the configuration?

Reply:
Yes, good catch.

Reply:
Resolved on line 195?